File: problems/3330-find-the-original-typed-string-i/solution.py
class Solution:
    def possibleStringCount(self, word: str) -> int:
        n = len(word)
        count = 1
        i = 0
        while i < n:
            j = i
            while j < n and word[i] == word[j]:
                j += 1
            if j - i > 1:
                count += j - i - 1
            i = j
        return count

File: problems/3330-find-the-original-typed-string-i/solution.java
class Solution {
    public int possibleStringCount(String word) {
        int count = 1;
        int n = word.length();
        for (int i = 0; i < n - 1; i++) {
            if (word.charAt(i) == word.charAt(i + 1)) {
                int j = i + 1;
                while (j < n && word.charAt(i) == word.charAt(j)) {
                    j++;
                }
                count += (j - i - 1);
                i = j - 1;
            }
        }
        return count;
    }
}